source/_docs/caching-advanced-topics.md

```diff
@@ -34,6 +34,12 @@ For more information, see [PANTHEON_STRIPPED GET Parameter Values](/docs/pantheo
 
 If your site or application requires Facebook authentication, we have added exceptions for this to allow users to register and log in. In the event you are having problems with another external authentication service, please contact us and let us know what service you are having issues with.
 
+## Manually Expiring Cache for Static Assets (e.g. CSS, JS, PDFs, Images)
+
+Pantheon sets a max-age header of 366 days on static assets. While these can be expired from the Pantheon Edge, an end user's browser or a CDN in front of Pantheon will cache the asset for a year - something neither site owners, site developers, or Pantheon can control.
```

Comment:
> something neither site owners, site developers, or Pantheon can control.
We've actually set this long expiration period from the VCL. We should remove the Pantheon part.

Reply:
@alexdicianu I think the point is that we can't control the behavior of the browser.